### PR TITLE
Feature/86cuwgjpe  display review object

### PR DESCRIPTION
**Pull request by hiri45**
https://app.clickup.com/t/86cuwgjpe
artist reviews tab represents each review object (i.e. the review comment)

### CODE REVIEW

File: behind-the-veil-siteroot/imports/ui/components/DatabaseHelper.jsx [hunk 1]
```diff
@@ -471,6 +471,7 @@ export function useGalleryTotalCollection(username) {
 }
 
 
+
```
Comment (nethbtju) [inline behind-the-veil-siteroot/imports/ui/components/DatabaseHelper.jsx:474]: can you remove these random lines?

File: behind-the-veil-siteroot/imports/ui/components/DatabaseHelper.jsx [hunk 2]
```diff
@@ -511,13 +512,59 @@ export function useUserBookings(username) {
...
+    const service = serviceArray.find(service => service._id === booking.serviceId);
+
+    return {
+      ...review,
+      booking,
+      service,
+    };
+  });
+
+console.log(reviewArray);
```
Comment (nethbtju) [inline behind-the-veil-siteroot/imports/ui/components/DatabaseHelper.jsx:564]: remove console.log

File: behind-the-veil-siteroot/imports/ui/components/pages/profile/artistTabs/ArtistReviewsTab.jsx [hunk 3]
```diff
@@ -3,21 +3,11 @@ import Loader from '../../../loader/Loader';
...
-                loadingText={"Reviews are loading . . ."}
-                isLoading={isLoading}
-                size={100}
-                speed={1.5}
-            />
-        )
-    }
+    const { isLoading, reviewArray, artistReviewData } = useArtistReviews(username);
 
+    console.log(reviewArray);
```
Comment (nethbtju) [inline behind-the-veil-siteroot/imports/ui/components/pages/profile/artistTabs/ArtistReviewsTab.jsx:10]: remove console.log

File: behind-the-veil-siteroot/imports/ui/components/pages/profile/artistTabs/ArtistReviewsTab.jsx [hunk 4]
```diff
@@ -51,59 +41,81 @@ export const ArtistReviewsTab = ({ username }) => {
...
+    if (isLoading) {
+        return (
+            <Loader
+                loadingText={"Reviews are loading . . ."}
+                isLoading={isLoading}
+                size={100}
+                speed={1.5}
+            />
+        )
+    } else {    console.log(reviewStats);
```
Comment (nethbtju) [inline behind-the-veil-siteroot/imports/ui/components/pages/profile/artistTabs/ArtistReviewsTab.jsx:53]: probably don't need this in develop branch

**Review by nethbtju — CHANGES_REQUESTED** [review]
What did you use to test this? was it mock data or actual bookings?

**Comment by hiri45** [comment]
> What did you use to test this? was it mock data or actual bookings?

i used mock data

File: behind-the-veil-siteroot/imports/ui/components/card/ReviewCard.jsx [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+ * @param {string} price - Price of service
+ * // TODO: may need to refactor props once the database schema is set up
+ */
+const ReviewCard = ({ className,reviewTitle, reviewComment, date, service, price }) => {
+  const classes = classNames("review-details", className)
+  return (
+    <Card className={classes}>
+      <div>
+        {/*<div className="main-text text-dark-grey">Service</div>*/}
+        <div className="title-text text-our-black">Review Title{reviewTitle}</div>
```
Comment (nethbtju) [inline behind-the-veil-siteroot/imports/ui/components/card/ReviewCard.jsx:27]: this shouldn't be using title text it doesn't match the figma

File: behind-the-veil-siteroot/imports/ui/components/card/ReviewCard.jsx [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+        <div className="main-text text-our-black max-w-xl break-words">{reviewComment}</div>
+      </div>
```
Comment (nethbtju) [inline behind-the-veil-siteroot/imports/ui/components/card/ReviewCard.jsx:29]: should have a horizontal line down the middle of the whole card

File: behind-the-veil-siteroot/imports/ui/components/card/ReviewCard.jsx [hunk 7]
```diff
@@ -0,0 +1,40 @@
...
+      <div>
+        <div className="main-text text-dark-grey">Service: {service}</div>
+        <div className="main-text text-dark-grey">Price: ${price}</div>
+        <div className="main-text text-dark-grey">  Date: {date ? date.split('T')[0] : type ? type : ""}</div>
```
Comment (nethbtju) [inline behind-the-veil-siteroot/imports/ui/components/card/ReviewCard.jsx:33]: positioning of this is a bit off from the actual figma design + Missing AC 2 with the individual rating stars for each card

**Review by nethbtju — CHANGES_REQUESTED** [review]
Minor UI changes and missing AC2

File: behind-the-veil-siteroot/imports/ui/components/card/ReviewCard.jsx (hunk 5 above)
Comment (hiri45) [reply behind-the-veil-siteroot/imports/ui/components/card/ReviewCard.jsx:27]: what font am i meant to use?

File: behind-the-veil-siteroot/imports/ui/components/card/ReviewCard.jsx (hunk 6 above)
Comment (hiri45) [reply behind-the-veil-siteroot/imports/ui/components/card/ReviewCard.jsx:29]: i added the horizontal line however I have no idea why its so close to the other div would you have any idea on how to move it more towards the left?: https://tailwindcss.com/docs/divide-width

**Pull request merged**